Makefile

```diff
@@ -389,16 +389,14 @@ ci-coverage-report: $(SUMMARY_COVER_PROFILE) coverage-report
 	@goveralls -coverprofile=$(SUMMARY_COVER_PROFILE) -service=buildkite || true
 
 ##### Schema #####
-install-schema: temporal-cassandra-tool
+install-schema: install-schema-postgresql12
```

Comment:
It cannot run Cassandra by itself because it won't support Visibility anymore.
I don't know. I just picked one. I'm fine with Cassandra with Elasticsearch as well, or MySQL.